requirements-frozen.txt

```diff
@@ -1,353 +1,120 @@
-#
```

Comment:
What's the difference between `requirements.txt` and `requirements-frozen.txt`? Is this file in use? or is it preparatory work?

Reply:
you're commenting on an older commit which included a merge error